script/Arcueid/Day 8/08_00_ARC08_6B.txt

```diff
@@ -481,12 +481,12 @@ But that thing is different.
 [sha:f867c0a1e086983b74dab8f8f952107cd9fe1c94]{
 -- Page 385, Offset 20642.
 -- 空気が違う。種別が違う、存在にいたる工程が違っている。
-It's air is different. The species that it came from is different. The process for which it came into being is different.
+It's atmosphere is different. The species that it came from is different. The process for which it came into being is different.
 }
 [sha:6d29db00e0b820745027c5bc20895e309b4ff620]{
 -- Page 385, Offset 20643.
 -- 俺たちの街にも、アルクェイドの世界にも、いる筈のない“異種”だった。
-That thing is a different organism that shouldn't exist in our city, and even in the world of Arcueid.
+That thing is a different organism that shouldn't exist in our city, and even in Arcueid's world.
```

Comment:
That thing is an organism that shouldn't exist. Not in our city. Not even in Arcueid's world.